melodic/distribution.yaml

```diff
@@ -15380,6 +15380,21 @@ repositories:
       url: https://github.com/clearpathrobotics/wireless.git
       version: master
     status: maintained
+  witmotion_ros:
```

Comment:
The key here is `witmotion_ros`, and the package.xml in the sources say `witmotion_ros`, but the repository name is `witmotion_IMU_ros`.  Which one is correct?  Can we make these all the same?

Reply:
The repository is maintained by Elettra so it was un upstream decision to choose Github repository name. I will ask for renaming but it will require time (unknown amount, I see). The correctname is definitely `witmotion_ros` and it will never be changed, so here we can refer to this package name.